datasetinsights/commands/convert.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+import click
+
+import datasetinsights.constants as const
+from datasetinsights.datasets.transformers import COCOTransformer
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(context_settings=const.CONTEXT_SETTINGS)
+@click.option(
+    "-i",
+    "--input",
+    type=click.Path(exists=True, file_okay=False),
+    required=True,
+    help="Directory of the Synthetic dataset.",
+)
+@click.option(
+    "-o",
+    "--output",
+    type=click.Path(file_okay=False, writable=True),
+    required=True,
+    help="Directory of the converted dataset.",
+)
+@click.option(
+    "-f",
+    "--format",
+    required=True,
+    help=("The output dataset format. Currently only COCO is supported."),
+)
+# TODO(YC) We need to figure out a better way to supply definition ID here
+# in order to support other annotations like instance segmentation or keypoints.
+# Ideally, we should be able to detect all available annotations that's
+# compatible to COCO formats.
+@click.option(
```

Comment:
@saad-unity says: When converting data from perception to coco, why can’t the --bbox2d-definition-id be automatically read from annotation_definitions.json?

Reply:
I think maintaining the RGB filename to be the same as the original file would be better. Currently the UUID is converted to int and the filename is written as f"camera_{capture_id}.png"

Reply:
I will implement `annotation_definition_id` lookup to automatically find the right `annotation_id` for 2D bounding boxes. 

The second question is answered separately in a different thread.

Reply:
added.